editor/script_create_dialog.cpp

```diff
@@ -373,7 +373,7 @@ void ScriptCreateDialog::_create_new() {
 	const ScriptLanguage::ScriptTemplate sinfo = _get_current_template();
 
 	String parent_class = parent_name->get_text();
-	if (!ClassDB::class_exists(parent_class) && !ScriptServer::is_global_class(parent_class)) {
+	if (!parent_name->get_text().begins_with("\"res://") && !ClassDB::class_exists(parent_class) && !ScriptServer::is_global_class(parent_class)) {
```

Comment:
How does it behave when using `res://script.gd` without quotes?

Reply:
It's invalid
![image](https://user-images.githubusercontent.com/2223172/190424486-62527bcc-025c-4267-9f4a-32c46b9875fc.png)

Reply:
Ah, but it doesn't handle relative paths. Maybe using quote only is ok 🤔

Reply:
BTW you can reuse `parent_class` instead of `parent_name->get_text()`.

Reply:
`parent_class.is_quoted()`

Reply:
Ok fixed. It relies on the path being validated properly, but it's *probably* ok.